packages/client/src/loadConfiguration.ts

```diff
@@ -1,15 +1,21 @@
 import Configuration from './configuration';
 
 export const DefaultURL = 'https://app.land';
+export const DefaultApiURL = 'https://api.getappmap.com';
 
 class Settings {
   baseURL = DefaultURL;
+  apiURL = DefaultApiURL;
   apiKey?: string;
 }
 
 function loadFromEnvironment(): Settings {
   const settings = new Settings();
 
+  ['APPLAND_API_URL', 'APPMAP_API_URL'].forEach((key) => {
```

Comment:
Do we want new env vars for these? If `APPMAP_URL` is `getappmap.com` then `API_URL` can be `api.getappmap.com`